inductiva/tasks/task.py

```diff
@@ -35,13 +35,50 @@ class Task:
         _api: Instance of TasksApi (from the generated API client).
     """
 
+    FAILED_STATUSES = {
+        models.TaskStatusCode.FAILED, models.TaskStatusCode.KILLED,
+        models.TaskStatusCode.EXECUTERFAILED,
+        models.TaskStatusCode.EXECUTERTERMINATED,
+        models.TaskStatusCode.EXECUTERTERMINATEDBYUSER,
+        models.TaskStatusCode.SPOTINSTANCEPREEMPTED,
+        models.TaskStatusCode.ZOMBIE
+    }
+
+    TERMINAL_STATUSES = {models.TaskStatusCode.SUCCESS}.union(FAILED_STATUSES)
+
+    RUNNING_STATUSES = {
+        models.TaskStatusCode.PENDINGINPUT, models.TaskStatusCode.SUBMITTED,
+        models.TaskStatusCode.STARTED
+    }
+
     def __init__(self, task_id: str):
         """Initialize the instance from a task ID."""
         self.id = task_id
         self._api = tasks_api.TasksApi(api.get_client())
         self._info = None
         self._status = None
 
+    def is_running(self) -> bool:
+        """Validate if the task is running.
+
+        This method issues a request to the API.
+        """
+        return self.get_status() in self.RUNNING_STATUSES
```

Comment:
this is not the same as the previous implementation

Reply:
I reverted to the previous implementation.